Library/Formula/vobcopy.rb

```diff
@@ -14,6 +11,11 @@ def install
                              "--with-lfs",
                              "--with-dvdread-libs=#{HOMEBREW_PREFIX}"
     system "make"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    output = shell_output("#{bin}/vobcopy -h 2>&1", 1)
```

Comment:
I found [some VOB files at digital-digest.com](http://www.digital-digest.com/movies/movie_index.php?type=dolby) that look promising.